specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json

```diff
@@ -0,0 +1,1119 @@
+          "nextLinkName": "nextLink",
+          "itemName": "models"
+        }
+      }
+    },
+    "/multivariate/models/{modelId}": {
+      "delete": {
+        "responses": {
+          "204": {
+            "description": "Successfully marked model for deletion. Model will be removed within a predefined time period."
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "Error code",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Delete Multivariate Model",
+        "description": "Delete an existing multivariate model according to the modelId",
+        "operationId": "DeleteMultivariateModel",
+        "parameters": [
+          {
+            "name": "modelId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Model identifier",
+            "format": "uuid"
+          }
+        ],
+        "x-ms-examples": {
+          "Delete multivariate model": {
+            "$ref": "./examples/DeleteModel.json"
+          }
+        }
+      },
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Get multivariate model details successfully.",
+            "schema": {
+              "$ref": "#/definitions/Model"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "Error code",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Get Multivariate Model",
+        "description": "Get detailed information of multivariate model, including the training status and variables used in the model.",
+        "operationId": "GetMultivariateModel",
+        "parameters": [
+          {
+            "name": "modelId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Model identifier",
+            "format": "uuid"
+          }
+        ],
+        "x-ms-examples": {
+          "Get multivariate model": {
+            "$ref": "./examples/GetModel.json"
+          }
+        }
+      }
+    },
+    "/multivariate/models/{modelId}/entire/detect": {
+      "post": {
+        "responses": {
+          "201": {
+            "description": "Submit a multivariate model detection task successfully.",
+            "headers": {
+              "Location": {
+                "description": "Location and ID of the detection result.",
+                "type": "string"
+              }
+            }
+          },
```

Comment:
This looks like a long-running POST action operation. We recently updated our guidelines on LROs and would really like you to adopt the new pattern. Please see [this section](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations--jobs).

Reply:
The response code should be 200 because this does not create a resource. The response header should be "Operation-Location".  You could continue to return "Location" if necessary for backward compat with your preview if business needs require it.

Reply:
Fixed.